### PR TITLE
feat: disable kbcli addons in batch

### PR DESCRIPTION
**Pull request by SuperAayush**
The PR contains the code to make the kbcli to disable more than one addon at once.

Fixes: #237

### CODE REVIEW

File: addons
Comment (shanshanying) [inline addons:1]: do not commit changes of submodules.

**Comment by 1aal** [comment]
Hi @SuperAayush , nice to receive your PR! 
I personally have some thoughts about your PR, I think there are some object that we don't need to repeat the initialization such as `dynamic.Interface` in `addonCmdOpts`.
Another question is what happens if, in the process of disabling a bunch of addons, one of them fails to be disabled. Like
`kbcli addon disable apecloud-mysql WRONG redis`, we expect the `apecloud-mysql` and `redis` to successfully disable and for the`WRONG` , it will report an error. Or neither of them will be disabled, with the corresponding error message

**Comment by SuperAayush** [comment]
Hey @1aal, I agree with your suggestions I had a chat regarding the same with @shanshanying over a call.
While adding the test case I ran into multiple errors and debugged them as well, since there is no working test case for the enable command I can't debug the complete process. 
The code block that would trigger the test is commented: https://github.com/apecloud/kbcli/blob/7dc25568a9cf6e5555e1af1fd78c67d0b8e8de20/pkg/cmd/addon/addon_test.go#L114

Also after validating the test, the process is stuck here: https://github.com/apecloud/kbcli/blob/7dc25568a9cf6e5555e1af1fd78c67d0b8e8de20/pkg/cmd/addon/addon.go#L332

I have even tried to fetch the selector labels and they match exactly but the process is still stuck.
Happy to connect with you over a call to debug this.

cc - @shanshanying 

Thanks!!

**Review by ldming — APPROVED** [review]

**Comment by nayutah** [comment]
@SuperAayush this pr has been refined and merged to main branch https://github.com/apecloud/kbcli/commit/a97d12d08bf85709a29b719bfdcdecb5026452cd

**Pull request closed without merging**